test cases/common/278 extract_all custom_target/subprojects/s1/meson.build

```diff
@@ -0,0 +1,26 @@
+project('subproj', 'c')
+
+fs = import('fs')
+copy = find_program('../../copy.py')
```

Comment:
This shouldn't work, you should be getting a sandbox error (I think)

What I would do to make this work is in your root project do:
```meson
copy = find_program('copy.py')
meson.override_find_program('copy', copy)
```

and then in the subprojects you can do `find_program('copy')`, and that will take the override.

Reply:
Thanks, applied